Classes/Utility/ImageUtility.php

```diff
@@ -259,13 +259,17 @@ protected static function getBasicMetadata(string $fileName): array
      */
     protected static function safeUtf8Encode(?string $text): ?string
     {
+        if (function_exists('mb_convert_encoding') === false) {
```

Comment:
the whole point of that method is to support some "legacy" if `mb_convert_encoding` respectively `mb_detect_encoding` which is part of the same module is not available. Thus returning right away makes no sense.